### DISCUSSION TITLE
How to load sp components from CDN?

### DISCUSSION THREAD

**Discussion by openscript** [adobe/spectrum-web-components#3705] [Q&A]
I want to load some components from Spectrum via the proposed CDN. Unfortunately the loaded components aren't optimized for production: 

> Lit is in dev mode. Not recommended for production! See https://lit.dev/msg/dev-mode for more information.

I've created a minimal reproduction:

https://codepen.io/openscript/pen/gOZQeXp

How to load spectrum components in production with buildless environment?

The suggested CDN doesn't work well in the country I'm residing in. I need to use jsdelivr. How can I get Spectrum components to work with jsdelivr?

I've created another minimal reproduction:

https://codepen.io/openscript/pen/eYbQMMr

**Comment by Westbrook** [comment]
You need to work with jsdeliver and find a way for them to surface export conditions on their ESM output. CDNs like JSPM offer this to ensure the production code is served when desired. In that this is a third party offering, there’s little we can do to support you in this effort.

**Comment by steveswe** [comment]
I was also not successful using `jsdelivr`, the code the website suggests does not work:

https://www.jsdelivr.com/package/npm/@spectrum-web-components/bundle
